src/main/java/com/mrbysco/disccord/client/audio/FFmpeg.java

```diff
@@ -109,9 +111,9 @@ static void checkForExecutable() throws IOException {
 	 * Checks if the 'ffmpeg' executable is in the path
 	 * @return Whether the 'ffmpeg' executable is present
 	 */
-	static boolean checkFFMpegPath() {
+	static boolean checkFFMpegPath(String path) {
```

Comment:
Any reason for adding this argument when it's always `ffmpeg`?

Reply:
It happens on line 144 of FFmeg.java to make sure `ffmpeg` is still executable. The previous behaviour was to check if the filepath existed, but that didn't work for binaries in the `PATH`.

Reply:
Ah ok 👍